components/profile_picture.tsx

```diff
@@ -51,6 +52,11 @@ export default class ProfilePicture extends React.PureComponent<Props> {
             this.props.src;
 
         const profileIconClass = `profile-icon ${this.props.isEmoji ? 'emoji' : ''}`;
+        let statusIcon = <StatusIcon status={this.props.status}/>;
+
+        if (this.props.statusHidden) {
+            statusIcon = <></>;
+        }
```

Comment:
Instead of `<></>`, we normally do the other way, something like:
```suggestion
        let statusIcon;
        if (!this.props.statusHidden) {
            statusIcon = <StatusIcon status={this.props.status}/>;
        }
```